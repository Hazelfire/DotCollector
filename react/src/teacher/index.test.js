import React, {Component} from 'react';
import renderer from 'react-test-renderer';
import Teacher from './index';

window.fetch = () =>{
  return {"then": window.fetch}
};

window.Headers = () =>{

};

test('renders the teacher screen', () => {
  const component = renderer.create(<Teacher />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
