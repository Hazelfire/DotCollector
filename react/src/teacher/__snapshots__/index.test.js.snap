// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the teacher screen 1`] = `
<div
  className="teacher-background"
>
  <div
    className="header-bar"
  >
    <div
      className="ui inverted teacher-background menu"
    >
      <div
        className="item"
      >
        <img
          className="ui mini image"
          src="test-file-stub"
        />
      </div>
      <div
        className="item"
      >
        <div
          className="ui header"
        >
          Dot Collector
        </div>
      </div>
    </div>
  </div>
  <div>
    <button
      className="ui button"
      disabled={undefined}
      onClick={[Function]}
      role="button"
      tabIndex={undefined}
    >
      New Session
    </button>
    <div
      className="session-listing"
    >
      Loading
    </div>
  </div>
</div>
`;
